x-pack/legacy/plugins/fleet/public/pages/agent_list/components/agent_enrollment.tsx

```diff
@@ -23,7 +23,6 @@ import {
 } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n/react';
 import React, { Fragment, useState } from 'react';
-import { Policy } from '../../../../common/types/domain_data';
```

Comment:
I don't think this import needs to be changed, as the path was [updated already](https://github.com/elastic/kibana/pull/52061/files#diff-b834b42580f51b467e14fd3108203566R7)